db/sql/0_msar.sql

```diff
@@ -101,10 +101,22 @@ $$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
 ----------------------------------------------------------------------------------------------------
 -- INFO FUNCTIONS
 --
--- Functions in this section get information about a given table or column.
+-- Functions in this section get information about a given schema, table or column.
 ----------------------------------------------------------------------------------------------------
 ----------------------------------------------------------------------------------------------------
 
+CREATE OR REPLACE FUNCTION __msar.get_sch_name(sch_id oid) RETURNS TEXT AS $$/*
```

Comment:
I prefer to write out `schema` in the function name (but not variable names).

It's a bit picky, but we don't really need to save horizontal space on function names, just variables and function bodies.